packages/api/src/command/medical/organization/create-organization.ts

```diff
@@ -37,9 +41,41 @@ export const createOrganization = async (
   const fhirOrg = toFHIR(org);
   await upsertOrgToFHIRServer(org.cxId, fhirOrg);
 
-  if (org.type !== "healthcare_it_vendor") {
+  if (org.type === "healthcare_provider") {
+    // TODO Move to external/hie
```

Comment:
let's tie any TODOs in the code to a specific issue number

Reply:
Done.